pkg-r/R/deprecated.R

```diff
@@ -0,0 +1,158 @@
+#' Deprecated functions
+#'
+#' These functions have been replaced by the new `QueryChat` R6 class API.
+#' Please update your code to use the new class-based approach.
+#'
+#' @name deprecated
+#' @keywords internal
+NULL
+
+#' @rdname deprecated
+#' @export
+querychat_init <- function(...) {
+  lifecycle::deprecate_stop(
+    when = "0.1.0",
+    what = "querychat_init()",
+    with = "QueryChat$new()",
```

Comment:
This line made me think that I wouldn't be opposed to having a helper function that creates a new `QueryChat` instance. My biggest complaint with `R6` classes is that it is very hard as a user to find docs about the most important step: creating a new instance of the class.

You can't inspect `$new()` to see the arguments and you don't get any help from autocomplete (IIRC). `?QueryChat` often is overfull with context, especially when all you need is a little help to get started.

I'd be okay with `querychat()` as a direct interface to `QueryChat$new()`.